src/app/stack/card-details/card-details.component.ts

```diff
@@ -87,7 +87,7 @@ export class CardDetailsComponent implements OnChanges {
     public titleAndDescription: any = {
         [this.cardTypes.SECURITY]: {
             title: 'Dependencies with security issues in your stack',
-            description: 'A list of the dependencies affected with common vulnerabilities and exposures (CVE), dependency with the highest common vulnerability score (CVSS), and its CVE ID. You can take corrective actions by reporting the issues.'
+            description: 'A list of the dependencies affected with common vulnerabilities and exposures (CVE), dependency with the highest common vulnerability score (CVSS), and its Vulnerability ID.'
```

Comment:
'A list of the dependencies affected with common vulnerabilities and exposures (CVE) and vulnerabilities unique to Snyk, dependency with the highest common vulnerability score (CVSS), and its Vulnerability ID.'

Reply:
@dvandra we can add the statement "vulnerabilities unique to Snyk" in the description, but we have exactly the same there in the 2nd tab(title) of Security Card. It may be redundant. It would be great if we could reframe it into something else.